ebpf/state.go

```diff
@@ -320,49 +304,23 @@ func (ns *networkState) RemoveClient(clientID string) {
 	delete(ns.clients, clientID)
 }
 
-func (ns *networkState) cleanupState(now time.Time, clearExpiredStats, flushStats bool) {
+func (ns *networkState) RemoveConnections(keys []string) {
 	ns.Lock()
 	defer ns.Unlock()
 
-	// Remove expired clients + stats
-	deletedStats := 0
+	now := time.Now()
+
+	// Remove connections
 	for id, c := range ns.clients {
-		if c.lastFetch.Add(ns.expiry).Before(now) {
+		// Check if we should expire client
+		if c.lastFetch.Add(ns.clientExpiry).Before(now) {
```

Comment:
If no connections are ever expired, then this will never get called... right? Not sure we want to do that

Reply:
They were still getting expired even if `keys` was empty but I will make a separate function 👍